vars/createGithubIssue.groovy

```diff
@@ -76,6 +76,19 @@ void call(Map args = [:]) {
             }
             else {
                 println("Creating new issue")
+                def labelName = sh(
+                        script: "gh label list --repo ${args.repoUrl} -S ${i} --json name --jq '.[0].name'",
+                        returnStdout: true
+                    ).trim()
+                if (labelName.equals(i.trim())) {
+                    println("Label ${i} already exists. Skipping label creation")
+                } else {
+                    println("${i} label is missing. Creating the missing label")
+                    sh(
+                        script: "gh label create ${i} --repo ${args.repoUrl}",
```

Comment:
I wonder if we should use `force` in case someone create the label just right after your check.

Reply:
Thats a very rare race condition of 1 sec, I am okay to see how this works. If it happens too much we can add force as an enhancement. Let me know what you think. Just not liking the idea of forcing overwriting of something.